src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -246,7 +248,7 @@ func (c *controller) waitAndUpdateNodesStatus() bool {
 		}
 		common.LogIfHostIpChanged(c.log, node, knownIpAddresses)
 
-		if host.Host.Progress.CurrentStage == models.HostStageConfiguring {
+		if funk.Contains([]models.HostStage{models.HostStageConfiguring, models.HostStageRebooting}, host.Host.Progress.CurrentStage) {
```

Comment:
See https://issues.redhat.com/browse/MGMT-12303

Reply:
In particular:

>Also we want to introduce new stages with the new reporter service, and so being strict with requiring Configuring will interfere with that 

i.e. I think we should remove this condition altogether, and not be strict about the particular stage, as we might introduce the reporter soon and that means adding new stages which would have to be added to this `if` condition

Reply:
Not sure what is reporter and why we are going to add new stages, we still need to know when to send joined and when to send done, I don't want to send something each loop run.
So it will fix [MGMT-12303](https://issues.redhat.com/browse/MGMT-12303)

Reply:
>Not sure what is reporter

https://github.com/openshift/assisted-service/blob/master/docs/enhancements/boot-status-reporter.md

Reply:
It is not going to add any status or update installation flow, will be used for logs upload for now at least

Reply:
The proposal suggests adding new progress stages

Reply:
Anyway we'll just have to remember to update this condition when those new stages are added, that's all. Or find some way to get rid of this condition without having to list stages